src/communication/network_interface_sr.c

```diff
@@ -10223,3 +10226,209 @@ ssession_stop_attached_threads (void *session)
 {
   session_stop_attached_threads (session);
 }
+
+void
+scdc_find_lsa (THREAD_ENTRY * thread_p, unsigned int rid, char *request, int reqlen)
+{
+  OR_ALIGNED_BUF (OR_INT_SIZE + OR_LOG_LSA_ALIGNED_SIZE + OR_INT64_SIZE) a_reply;
+  char *reply = OR_ALIGNED_BUF_START (a_reply);
+  char *ptr;
+  LOG_LSA start_lsa;
+  time_t input_time;
+  uint64_t b_start_lsa;
+  int error;
+
+  ptr = or_unpack_int64 (request, &input_time);
+
+  error = cdc_find_lsa (thread_p, &input_time, &start_lsa);
+
+  ptr = or_pack_int (reply, error);
+  ptr = or_pack_log_lsa (ptr, &start_lsa);
+  or_pack_int64 (ptr, input_time);
```

Comment:
My guess is that input_time is semantically incorrect. Therefore, it is recommended to modify the variable name of input_time according to the meaning.

Reply:
Do we need to send additional information(start_lsa, imput_time) in case of an error?